src/models/tag/stream.rs

```diff
@@ -1,19 +1,22 @@
+use crate::db::graph::exec::retrieve_from_graph;
+use crate::db::kv::index::sorted_sets::SortOrder;
 use crate::types::DynError;
 use axum::async_trait;
-use neo4rs::Query;
-use serde::de::DeserializeOwned;
+use chrono::Datelike;
 use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::fmt::Display;
 use std::ops::Deref;
 use utoipa::ToSchema;
 
-use crate::db::kv::index::sorted_sets::SortOrder;
-use crate::models::follow::{Followers, Following, Friends, UserFollows};
-use crate::{db::connectors::neo4j::get_neo4j_graph, queries};
+use crate::queries;
 use crate::{RedisOps, ScoreAction};
 
 pub const TAG_GLOBAL_HOT: [&str; 3] = ["Tags", "Global", "Hot"];
 
-#[derive(Serialize, Deserialize, Debug, ToSchema)]
+const GLOBAL_HOT_TAGS_PREFIX: &str = "Hot_Tags";
```

Comment:
Lets follow the convention that we do, all the indexes has to be constant and names separated by `:`. An option or you can do something similar:
```rust
// SORTED SET, global. Use TAG_GLOBAL_HOT
// INDEX: Sorted:Tags:Global:Hot
pub const TAG_GLOBAL_HOT: [&str; 3] = ["Tags", "Global", "Hot"];
// These ones, the prefix has to be `CACHE` because they have an expiration time
// SORTED SETs, timeline. 
// INDEX: Cache:Sorted:Tags:Today:Post
pub const TAG_HOT_TODAY: [&str; 3] = ["Tags", "Today", "Post"];
// INDEX: Cache:Sorted:Tags:ThisMonth:Post
pub const TAG_HOT_TODAY: [&str; 3] = ["Tags", "ThisMonth", "Post"];
// Taggers JSON
// INDEX: Cache:Tags:Today:Post:Taggers
pub const TAGGERS_HOT_TODAY: [&str; 3] = ["Tags", "Today", "Post", "Taggers"];
// INDEX: Cache:Tags:ThisMonth:Post:Taggers
pub const TAGGERS_HOT_THIS_MONTH: [&str; 3] = ["Tags", "ThisMonth", "Post", "Taggers"];
```